vendor/plugins/edu.harvard.dce.paella.handoutDownloadPlugin/handout.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * #DCE MATT-1794, UI pluging for user to access one or more Mediapackage attachments
+ * of type "attachment/notes". For example, a PDF handout.
+ * #DCE-1840, Change handout download to a single button click
+ */
+Class ("paella.plugins.handoutDownloadPlugin", paella.ButtonPlugin,{
+	_attachementPath: ['matterhorn', 'episode', 'mediapackage', 'attachments', 'attachment'],
```

Comment:
`_attachementPath` typo. I know this seems picky, but typos make it harder to search for things later.